Library/Formula/fish.rb

```diff
@@ -21,6 +21,9 @@ class Fish < Formula
 
   def install
     system "autoconf" if build.head?
+    inreplace "fish.pc.in" do |s|
+      s.gsub! /^completionsdir=.*$/, "completionsdir=#{HOMEBREW_PREFIX}/share/fish/vendor_completions.d"
```

Comment:
@mikemcquaid Please do not merge this. I have a discussion with fish maintainer, the `pkg-config` way maybe changed.